README.md

```diff
@@ -10,12 +10,10 @@
 
 ## Cats
 
-![cats image](http://plastic-idolatry.com/erik/cats2.png)
+![Cats graphic](https://user-images.githubusercontent.com/52038/121751434-05a6ec80-cb06-11eb-9b7e-d9f126a0a4e4.png)
```

Comment:
The Cats graphic at http://plastic-idolatry.com/erik/cats2.png used HTTP, so was considered insecure by modern browsers on the HTTPS web page https://typelevel.org/cats/ - leading to another broken image:
![image](https://user-images.githubusercontent.com/52038/121753010-4eac7000-cb09-11eb-80ce-4f0ad9875ee5.png)

Reply:
Should we put the Cats image into the repository? Or where does the `githubusercontent.com` address come from?

Reply:
`user-images.githubusercontent.com` is [the domain that GitHub uses](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/about-anonymized-urls) to serve the images that we [copy & paste into GitHub issues/PRs](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files):

![image](https://docs.github.com/assets/images/help/pull_requests/dragging_images.gif)

[A user can't change or delete an image](https://github.community/t/does-user-images-githubusercontent-expires/129292) once it's uploaded this way, and I believe you can think of them as being nearly as permanent as the existence of GitHub. I'm so lazy, I often copy-and-paste an image into an empty GitHub issue just to have somewhere public to serve the image from.

It's true that it's a rather opaque reference! If we were to put the image in the repo, whereabouts would it belong? All the current images in the project are in [`/docs/src/main/resources/microsite/img`](https://github.com/typelevel/cats/tree/main/docs/src/main/resources/microsite/img) - which gives urls that work for the microsite, eg https://typelevel.org/cats/img/sidebar_brand.png - should the ReadMe markdown refer to that address? An alternative would be the 'raw' GitHub url: https://raw.githubusercontent.com/typelevel/cats/main/docs/src/main/resources/microsite/img/sidebar_brand.png

Reply:
I think the raw URL is a good compromise, it should render correctly in both situations (website and README).

Reply:
Done with 26ba0d9ecc86fe1751d4205d94533b1d421e5a24 - the image will appear broken [in markdown preview](https://github.com/typelevel/cats/blob/26ba0d9ecc86fe1751d4205d94533b1d421e5a24/README.md#cats) until the PR is merged!

![image](https://user-images.githubusercontent.com/52038/121931928-248ec400-cd3c-11eb-8bc8-02771751e1db.png)